### PR TITLE
fix: exclude node_modules root

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
This PR contains a:

- [x] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

Exclude `node_modules` folder from root.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -72,9 +72,9 @@ class ESLintWebpackPlugin {
     const options = {
       ...this.options,
       exclude: parseFiles(
-        this.options.exclude || 'node_modules',
+        this.options.exclude || [],
         this.getContext(compiler)
-      ),
+      ).concat('**/node_modules/**'),
```
Comment (alexander-akait) [inline src/index.js:77]: I don't think it is good idea, sometimes developers need linting strange things, so if you provide own `exclude` you can setup any values, i.e. and enable linting `node_modules`

File: src/index.js [hunk 2]
```diff
@@ -72,15 +72,18 @@ class ESLintWebpackPlugin {
...
       extensions: arrify(this.options.extensions),
       files: parseFiles(this.options.files || '', this.getContext(compiler)),
     };
 
     const wanted = parseFoldersToGlobs(options.files, options.extensions);
-    const exclude = parseFoldersToGlobs(options.exclude, []);
+    const exclude = parseFoldersToGlobs(
+      this.options.exclude ? options.exclude : '**/node_modules/**',
+      []
+    );
```
Comment (alexander-akait) [inline src/index.js:86]: Just interesting why we have two `exclude`?

Comment (ricardogobbosouza) [reply src/index.js:86]: node_modules cannot be added with the context prefix

**Review by alexander-akait — APPROVED** [review]

File: src/index.js (hunk 2 above)
Comment (ricardogobbosouza) [reply src/index.js:86]: If the user sets the option to `exclude` and I do not add 'node_moules' otherwise it is added

**Pull request merged**